data/categories.txt

```diff
@@ -5565,35 +5565,35 @@ sk:Oceán
 sw:Bahari
 
 @wifi
-en:WiFi
-ru:WiFi
+en:WiFi|Wi-Fi
+ru:WiFi|Wi-Fi
 ar:وايفاي
```

Comment:
@jancborchardt I don't think we need to change the synonyms for these languages in this PR. I'm not sure if we have to change them at all.